home_position:
  # 7 dof arms [left_joint_1 left_joint_2, left_joint_3, left_joint_4, left_joint_5, left_joint_6, left_joint_7] 
  left_arm: [0.0, -0.4, 0.0, -1.9, 0.0, 1.1, 0.0] 
  right_arm: [0.0, 0.4, 0.0, 1.9, 0.0, -1.1, 0.0]

  # Index: left_joint_0_0, left_joint_1_0, left_joint_2_0, left_joint_3_0, 
  # Middle: left_joint_4_0, left_joint_5_0, left_joint_6_0, left_joint_7_0, 
  # Pinky: left_joint_8_0, left_joint_9_0, left_joint_10_0, left_joint_11_0, 
  # Thumb: left_joint_12_0, left_joint_13_0, left_joint_14_0, left_joint_15_0
  left_hand: [0.0, 0.0, 0.0, 0.0,
              0.0, 0.0, 0.0, 0.0, 
              0.0, 0.0, 0.0, 0.0, 
              0.0, 0.0, 0.0, 0.0] 
  right_hand: [0.0, 0.0, 0.0, 0.0,
              0.0, 0.0, 0.0, 0.0, 
              0.0, 0.0, 0.0, 0.0, 
              0.0, 0.0, 0.0, 0.0] 
default_move_duration: 3.0  # optional default

finger_grasp_speed: 0.2
finger_release_speed: 0.3